galaxy_ng/tests/integration/cli/test_dependencies.py

```diff
@@ -68,11 +70,17 @@ def test_collection_dependency_install(ansible_config, published, cleanup_collec
         else:
             raise
 
+    # wait for move task from `inbound-<namespace>` repo to `staging` repo
+    time.sleep(SLEEP_SECONDS_ONETIME)
+
     if retcode == 0:
         config = ansible_config("ansible_insights")
         client = get_client(config)
         set_certification(client, artifact2)
 
+        # wait for move task from `staging` repo to `published` repo
+        time.sleep(SLEEP_SECONDS_ONETIME)
```

Comment:
we have the method `wait_for_url`

`    dest_url = (
        f"v3/plugin/ansible/content/{level}/collections/index/"
        f"{collection.namespace}/{collection.name}/versions/{collection.version}/"
    )
    wait_for_url(client, dest_url)`

Reply:
we don't have `wait_for_url` in `stable-4.5`, it was added in 4.6, but I can try to add it to PR